The USES=libtool is adding dragonfly to "freebsd*)" in unwanted places.
This is an unintended side effect, block it with a hack for now.

--- configure.intermediate	2014-05-26 19:04:29.404255000 +0000
+++ configure
@@ -18564,7 +18564,7 @@ freebsd[45]*)
 $as_echo "#define C_BSD 1" >>confdefs.h
 
     ;;
-freebsd*)
+fre[e]bsd*)
     if test "$have_pthreads" = "yes"; then
 	THREAD_LIBS="%%PTHREAD_LIBS%%"
 	TH_SAFE="-thread-safe"
@@ -21775,7 +21775,7 @@ fi
 
 
 case "$host_os" in
-freebsd*)
+fre[e]bsd*)
 { $as_echo "$as_me:${as_lineno-$LINENO}: checking for kinfo_getvmmap in -lutil" >&5
 $as_echo_n "checking for kinfo_getvmmap in -lutil... " >&6; }
 if ${ac_cv_lib_util_kinfo_getvmmap+:} false; then :
